SaveEarth/Projects/SaveEarth/SaveEarth/Sources/UserInterface/Enum/Screen.swift

```diff
@@ -0,0 +1,27 @@
+//
+//  Screen.swift
+//  SaveEarth
+//
+//  Created by 김용우 on 8/25/24.
+//
+
+enum Screen {
+  case setting(SettingFeature.State)
+}
+
+extension Screen {
+
+  var key: String {
+    switch self {
+      case .setting:  return "setting"
```

Comment:
Swift 5.1 부터 `return` 을 생략할 수 있게 되었는데요. 혹시 아래와 같이 작성하는 것은 어떠실까요?
```suggestion
      case .setting:
        "setting"
```

Reply:
좋습니다!